i18n/en.json

```diff
@@ -48,7 +48,7 @@
   "activity_log.browser": "Browser: {browser}",
   "activity_log.firstTime": "First time active: {date}, {time}",
   "activity_log.lastActivity": "Last activity: {date}, {time}",
-  "activity_log.logout": "Logout",
+  "activity_log.logout": "Log Out",
```

Comment:
@AninditaBasu This was missed during review but the corresponding default message for each component should be updated too when this change.  In this case, it should change https://github.com/mattermost/mattermost-webapp/blob/master/components/activity_log_modal/components/activity_log.jsx#L200.  Would you be open to submit a follow up PR to all those affected?  thanks

Reply:
@saturninoabril 
Yes, absolutely. I have one PR open at the moment (https://github.com/mattermost/mattermost-webapp/pull/4649); once PR 4649 is merged, I'll submit a PR for the default messages affected by the changes to the menus and action buttons made by this merge 4555.

Reply:
Awesome, thanks @AninditaBasu! By the way, I requested change to that PR.

Quick tip:  Instead of working on master, you may create a new branch per PR so each PR/branch is isolated.